salt/environment_settings.yml

```diff
@@ -141,19 +149,22 @@ environments:
           edx-worker:
             number: 1
             type: t2.medium
-      sandbox:
+      sandbox-residential-live:
         business_unit: operations
         domains:
-          cms: studio-micromasters.d.mitx.mit.edu
-          lms: micromasters.d.mitx.mit.edu
-          preview: preview-micromasters.d.mitx.mit.edu
+          cms: studio-mitx-qa-sandbox.mitx.mit.edu
+          lms: mitx-qa-sandbox.mitx.mit.edu
+          preview: preview-mitx-qa-sandbox.mitx.mit.edu
+          gitreload: gr-qa-sandbox.mitx.mit.edu
```

Comment:
There's no need for a gitreload URL on the sandbox box because we won't be running that service there.